src/utils/pseudoRandom32CharHex.ts

```diff
@@ -1,7 +1,12 @@
 import { times } from 'lodash-es'
 
 export function pseudoRandom32CharHex(): string {
-  return times(32).map(pseudoRandomHex).join('')
+  return pseudoRandomCharHex(32)
+}
+
+export function pseudoRandomCharHex(length: number): string {
+  if (length < 1) return ''
+  return times(length).map(pseudoRandomHex).join('')
```

Comment:
Please revert these changes. It also fails our lint CI: https://app.circleci.com/pipelines/github/Scrivito/scrivito-portal-app/2613/workflows/2b810e59-fd30-4488-b274-9e104a20ff96/jobs/9660